src/main/bash/sdkman-uninstall.sh

```diff
@@ -34,6 +34,9 @@ function __sdk_uninstall {
 	if [ -d "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}" ]; then
 		__sdkman_echo_green "Uninstalling ${candidate} ${version}..."
 		rm -rf "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
+	elif [ -L "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}" ]; then
```

Comment:
I don't think this will work across all platforms, as git bash and cygwin does not have a notion of symlinks.